### PR TITLE
feat: add omnictl

### LINKED ISSUE #58: Add omnictl
Hi 👋🏼 

It would be awesome if omnictl was able to be installed and managed by Homebrew.

Thanks!

### PR DESCRIPTION
**Pull request by UnknownBlunders**
I've never written a homebrew formula so I just copied the talosctl formula and updated it to make omnictl available. I also updated the readme.

Resolves #58

### CODE REVIEW

**Comment by UnknownBlunders** [comment]
I sent in another commit to add omnictl support in the update.sh script. I also wrote about update.sh usage in the readme

**Comment by UnknownBlunders** [comment]
I just pushed another commit since there's been a new omnictl release since I opened this.

File: Formula/omnictl.rb [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Copyright Contributors to the omni project.
+# SPDX-License-Identifier: MPL-2.0
```
Comment (frezbo) [inline Formula/omnictl.rb:2]: I'm not sure if this is right, since omni has BUSL, @steverfrancis thoughts?

Comment (UnknownBlunders) [reply Formula/omnictl.rb:2]: Omni is, but it looked like omnictl is MPL-2 https://github.com/siderolabs/omni/blob/main/client/LICENSE

Comment (frezbo) [reply Formula/omnictl.rb:2]: nope, client is MPL, not omnictl https://github.com/siderolabs/omni/blob/main/cmd/omnictl/main.go#L1

Comment (frezbo) [reply Formula/omnictl.rb:2]: I'll confirm from business side before merging this

Comment (UnknownBlunders) [reply Formula/omnictl.rb:2]: Any update from the business side? I’m happy to push a fix to change it to BUSL. Also, what should the copyright line say?

Comment (frezbo) [reply Formula/omnictl.rb:2]: sorry, i forgot about this, it should be busl. While you're at it mind updating omni version also to https://github.com/siderolabs/omni/releases/tag/v0.31.1

Comment (frezbo) [reply Formula/omnictl.rb:2]: `# Copyright Sidero Labs, Inc.` should be good

Comment (UnknownBlunders) [reply Formula/omnictl.rb]: Just pushed a fix for the license and copyright. I updated the Omni version on Friday

**Review by frezbo — APPROVED** [review]

**Pull request merged**